README.rst

```diff
@@ -39,11 +39,22 @@ Installation
 ============
 
 Make sure you have Python 2.7 or Python 3.6 and recent version of MXNet.
-You can install ``MXNet`` and ``GluonNLP`` using pip:
+You can install ``MXNet`` and ``GluonNLP`` using pip.
+
+In particular, if you want the ``GluonNLP`` built on most recent ``MXNet`` release:
+
+::
+
+    pip install --pre --upgrade mxnet=='1.3.1'
```

Comment:
I'm not sure what you intend to achieve with this command. `--pre` is a flag for allowing pip to search for pre-releases, while `mxnet==1.3.1` freezes it to a particular release version.

Reply:
Shall we remove the --pre flags here?

Reply:
Addressed. thanks